__tests__/test-util.ts

```diff
@@ -62,4 +62,9 @@ export class TestTarget extends RpcTarget {
   returnNull() { return null; }
   returnUndefined() { return undefined; }
   returnNumber(i: number) { return i; }
+
+  echo(arg: any) {
+    return arg;
```

Comment:
So, there's an ownership problem here:
* Any stubs in an RPC's params are owned by the caller. They will be auto-disposed upon return.
* Returning a stub from an RPC transfers ownership to the caller.

[The rules are documented here.](https://github.com/cloudflare/capnweb?tab=readme-ov-file#automatic-disposal)

But this means that when you `return arg` here, the caller now has double-ownership.

What ends up happening is, the parameter is disposed upon return, but that means that the returned value is now invalid.

You can solve this by:

```suggestion
    return arg.dup();
```

I'll admit I'm not thrilled with this state of affairs. But I don't know what else we can do. It's important that args are auto-disposed upon return, because otherwise stubs that the callee wasn't expecting to receive will always be leaked. It's also important that returns transfer ownership, because otherwise it's difficult to dispose a stub _after_ returning it.

Maybe we need to expose a helper that walks an object tree and dups all the stubs, so that this function can be written in a way that doesn't assume anything about the type of `arg`.